packages/jsii-reflect/lib/docs.ts

```diff
@@ -60,17 +72,47 @@ export class Docs {
     return lowestStability(this.docs.stability, this.parentDocs?.stability);
   }
 
+  /**
```

Comment:
Nit: I don't think these doc strings are adding any new information.

Reply:
Okay -- in go, it's recommended to document all exported methods, and I believe the CDK also enforced documentation of public methods. It just seemed like we were being inconsistent with what methods were being documented.